roles/provision_vm/README.md

```diff
@@ -406,7 +406,7 @@ N/A
     Set the guest ID.
     This parameter is case sensitive.
     This field is required when creating a virtual machine, not required when creating from the template.
-    Valid values are referenced here: https://code.vmware.com/apis/358/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+    Valid values are referenced here: https://developer.vmware.com/apis/202/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
```

Comment:
There is same issue also on line 559, can we fix it as well?

Reply:
Fixed it as well, used that page https://developer.vmware.com/apis/1720/vsphere/    (vSphere Web Services API 8.0U2)